src/index.html

```diff
@@ -3,10 +3,350 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin = "anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class = "header">
+      <a href="#home" class="header__link">
+        <img
+          class = "header__img"
+          src="./images/logo.png"
+          alt="home"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#Apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#Laptops&Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item"><a href="#Gadgets" class="nav__link">Gadgets</a></li>
+
+          <li class="nav__item"><a href="#Tablets" class="nav__link">Tablets</a></li>
+
+          <li class="nav__item"><a href="#Photo" class="nav__link">Photo</a></li>
+
+          <li class="nav__item"><a href="#Video" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__elem card" data-qa="card">
+        <img
+          class="card__product-image"
+          src="images/imac.jpeg"
+          alt= "APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+        <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <p class="stars__star"></p>
+
+            <p class="stars__star"></p>
+
+            <p class="stars__star"></p>
+
+            <p class="stars__star"></p>
+
+            <p class="stars__star"></p>
+          </div>
```

Comment:
use semantic tag, every star is not a paragraph

Reply:
Not fixed